tap_github/schemas/issue_events.json

```diff
@@ -954,10 +954,166 @@
       ]
     },
     "performed_via_github_app": {
-      "type": [
-        "null",
-        "string"
-      ]
+      "type": ["null", "object"],
+      "properties": {
+        "id": {
+          "type": ["null", "integer"]
+        },
+        "slug": {
+          "type": ["null", "string"]
+        },
+        "node_id": {
+          "type": ["null", "string"]
+        },
+        "owner": {
```

Comment:
Is it have the same fields as the comment's owner field? If yes then we should move it to the common folder and use ref here.

Reply:
Added a shard schema for the same